crates/ark/src/modules/rstudio/commands.R

```diff
@@ -6,7 +6,14 @@
         "activateTerminal" = "workbench.action.terminal.focus",
         # This command includes untitled files in RStudio:
         "saveAllSourceDocs" = "workbench.action.files.saveAll",
-        "vcsRefresh" = "git.refresh",
+        # This command is a silent no-op in RStudio when there is no git repo:
```

Comment:
I think it *might* be worth noting the issue, in case anyone is tempted to mess with this:

```suggestion
        # https://github.com/posit-dev/positron/issues/2697
        # This command is a silent no-op in RStudio when there is no git repo:
```